packages/client/src/createMinimalClient.ts

```diff
@@ -70,9 +70,9 @@ export function createMinimalClient(
     "ontologyProvider"
   > as any;
 
-  return Object.assign(minimalClient, {
+  return Object.freeze(Object.assign(minimalClient, {
```

Comment:
Cool, we're doing this to make props non-configurable and non-writable right?

Reply:
Yeah, i dont think its needed but its a nice failsafe and we never reassign this stuff